### PR TITLE
Add ability to specify target for included transforms

### PR DESCRIPTION
**Pull request by kaicataldo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-  ],
-  plugins: [
-    require("babel-plugin-add-module-exports")
-  ]
-};
+  if (processEnv === 'test') {
+    config.plugins.push(require('babel-plugin-istanbul').default);
+  }
+
+  if (optionsEnv === 'browser') {
```
Comment (weikinhuang) [inline index.js]: i don't like this

Comment (weikinhuang) [inline index.js]: consider using a preset option. see @hzoo

**Comment by hzoo** [comment]
If you want to add badges

```
## license-to-fail [![npm](https://img.shields.io/npm/v/license-to-fail.svg)](https://www.npmjs.com/package/license-to-fail) [![travis](https://img.shields.io/travis/behance/license-to-fail/master.svg)](https://travis-ci.org/behance/license-to-fail)
```

File: index.js (hunk 1 above)
Comment (weikinhuang) [inline index.js]: a browser isn't a `NODE_ENV`, you more likely want to say include browser dependencies always in this repo. ie. `test & node` or `test & browser`

**Comment by mikesherov** [comment]
I think it's a bit weird to have 2 conflicting Boolean options controlling a single Boolean value. What should the behavior of `node: true, browser: true` be?

**Comment by kaicataldo** [comment]
That was my initial reservation (and why I initially implemented it as `env: "node"` or `env: "browser"`, but @weikinhuang pointed out that it could just be additive (like the `env` config in ESLint). I don't see the harm in that, at least for the moment. The `node` env doesn't actually enable anything at the moment anyway - just doesn't enable the browser-specific transformations.

That being said, if the two options do diverge and enable conflicting transformations in the future, we'd definitely have to change strategies...The two options I see are my initial implementation, only having one option (i.e. `{ node: true }` and defaulting to `browser` otherwise, or throwing an error when both are `true`.

Do you have a preference?

**Comment by mikesherov** [comment]
This is a mixed metaphor. In ESLint, both options are additive. In this program, both bools filter down to one. You're forced to address the conflict in code, but not in config. It's a mismatch.

**Comment by kaicataldo** [comment]
Sure, makes sense - I thought they were two separate paths initially too, but the suggestion to do it this way made sense to me too. @weikinhuang Your thoughts?

**Comment by weikinhuang** [comment]
That's what I meant yesterday, having them be additive should be fine.

**Comment by kaicataldo** [comment]
Correct me if I'm wrong, but it sounds like you're both advocating for making everything additive in both the config and internally. Will update the PR accordingly.

**Comment by kaicataldo** [comment]
Actually, I'm a bit confused. It currently is additive (with the default value being `brower: true`, but we don't have any Node specific transforms...

**Comment by weikinhuang** [comment]
basically, get rid of the `node` option for now as we're not using it, and just keep the `browser` option. We can add the `node` option later when we have node specific plugins

**Comment by kaicataldo** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -33,3 +33,18 @@ require("babel-core").transform("code", {
   presets: ["behance"]
 });
 ```
+
+## Options
+* `browser` (default) - Enable browser specific presets and transforms
+* `node` - Enable Node specific presets and transforms
```
Comment (hzoo) [inline README.md]: update here as well if node is remove

**Comment by mikesherov** [comment]
:+1:

**Pull request merged**